pep-3147.txt

```diff
@@ -5,6 +5,7 @@ Last-Modified: $Date$
 Author: Barry Warsaw <barry@python.org>
 Status: Final
 Type: Standards Track
+Topic: Packaging
```

Comment:
I'd exclude this -- it's more about the default behaviour of the import system, rather than a Packaging interoperability standard or so.